README.md

```diff
@@ -1,12 +1,25 @@
 # alfresco-build-tools
 
-[![Build Status](https://travis-ci.com/Alfresco/alfresco-build-tools.svg?branch=master)](https://travis-ci.com/Alfresco/alfresco-build-tools)
```

Comment:
the travis build is still there so badge removal should be reverted

Reply:
just moved lower, but maybe as I've seen elsewhere recently a table with a description at the top is the best, will do that